src/App.js

```diff
@@ -1,36 +1,97 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, {useEffect, useState} from 'react';
+// import './App.css';
 
-const data = [{name: 'Whatever', age: 5}, {name: 'Johnny', age: 345}];
+const base = 'http://localhost:3001/';
+
+const loadPeople = async onLoad => {
+  const response = await fetch(base + 'people');
+  onLoad(await response.json());
+};
+
+const App = () => {
+  const [people, setPeople] = useState([]);
+
+  const emailRef = React.createRef();
+  const passwordRef = React.createRef();
+  const deleteEmailRef = React.createRef();
+
+  const submitNewUser = async e => {
+    console.log("This button is pressed" + emailRef)
+    e.preventDefault();
+    await fetch(base + 'addPerson', {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({
+        email: emailRef.current.value,
+        password: passwordRef.current.value
+      })
+    });
+    // TODO: need to show errors if they happen
+    await loadPeople(setPeople);
+  };
+
+  //Function for button to delete a user
+
+  const deletePerson = async e => {
+    
+    console.log("This button is pressed" + deleteEmailRef)
+
+    e.preventDefault();
+    await fetch(base + 'deletePerson', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        email: deleteEmailRef.current.value
+      })
+    });
+
+    
+
+    // TODO: need to show errors if they happen
+    await loadPeople(setPeople);
+  };
+
+
+  useEffect(() => {
+    // second argument is [], so only do once
+    loadPeople(setPeople);
+  }, []);
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <h1>Hello PuddinGlove</h1>
-
-        <table>
-          <thead>
-            <th>Name:</th>
-            <th>Age:</th>
-          </thead>
-          <tbody>
-            {data.map(row => (
-              <tr key={row.name}>
-                <td>{row.name}</td>
-                <td>{row.age}</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
-      </header>
+    <div className="container">
+      <table className="table table-condensed">
+        <thead>
+          <tr>
+            <th>Id:</th>
+            <th>Email:</th>
+            <th>Password Hash</th>
+          </tr>
+        </thead>
+        <tbody>
+          {people.map(({id, email, password}) => (
+            <tr key={id}>
+              <td>{id}</td>
+              <td>{email}</td>
+              <td>{password}</td>
+              <td><form onSubmit={deletePerson}>
+                User to Delete: <input type="text" ref={deleteEmailRef} />
```

Comment:
Did you mean to have this repeat for each line? 
![image](https://user-images.githubusercontent.com/1922963/72192633-119bf580-33cc-11ea-9d44-18c6d7207d01.png)

Also, I'm not sure why there's a form with a text input in it. Instead of the form, we could do `<button onClick={deletePerson(id)}>Delete User</button>`

And then `deletePerson` would look like this:
```
const deletePerson = id => async e => {
  e.preventDefault();
  await fetch(base + 'deletePerson', {
    method: 'POST',
    headers: {'Content-Type': 'application/json'},
    body: JSON.stringify({id})
  });
  // TODO: need to show errors if they happen
  await loadPeople(setPeople);
};
```

Reply:
Sorry I didn't write more notes yesterday on these things.. I had it repeating because originally I had a button like you wrote. I did it the same way too, but for some reason it wasn't receiving `{id}` properly and sending it along. So I thought I would try to get it working with just basic functionality first (basically copying the same format we had for adding a user, so you could type in a user's name and delete them.

Reply:
I have the extension installed for eslint and the settings on. Not sure why it's not working. Once I fix the other problems I will look into it more. I didn't notice it wasn't making the proper changes before.

![esLintNotWorking](https://user-images.githubusercontent.com/48955070/72206657-42287180-3456-11ea-9833-aff39f6dead2.png)

Reply:
Ok. The linting isn't a big deal. Let's get the delete button working though.